docs/reference-use-cases.md

```diff
@@ -0,0 +1,215 @@
+
+### Removing Cluster API
+- As an operator of a management cluster, given that I have a management cluster that I have used to deploy several workload clusters, I want to remove the Cluster/Machine objects representing one workload cluster from my management cluster without deprovisioning workload cluster.
+
+- As an operator of a management cluster, given that I have a management cluster that I have used to deploy several workload clusters, I want to uninstall Cluster API from my management cluster without deprovisioning my workload clusters.
+
+- As an operator of a management cluster, given that I have a management cluster, I want to use it to manage workload clusters that were created by a different management cluster.
+
+### Cross-cluster Metrics
+- As an operator of a management cluster, I want to query my resource allocation on an infrastructure.  For example, in an on-prem case, I do not have an infinite capacity cloud, so  I need to be able to determine my reservation before deploying a workload cluster.
+
+### Specific Architecture Approaches
+- As an operator of a management cluster, given that I give operators of workload clusters access to my management cluster, they can launch new workload clusters with control planes that run in the management cluster while the nodes of those workload clusters run elsewhere.
+
+- As a multi-cluster operator, I would like to provide an EKS-like experience. Where the workload control plane nodes are joined to the management cluster and the control plane config isn’t exposed to the consumer of the workload cluster. This enables me as an operator to manage the control plane nodes for all clusters using tooling like prometheus and fluentd. I can also control the configuration of the workload control plane in accordance with business policy.
+
+### Multitenancy Management
+- As an operator of a management cluster, I want to control which users of management cluster can deploy new workload clusters, how many clusters they can deploy, and how many nodes/resources those clusters can use.
+
+- As an operator of a management cluster, I want to ensure that only the user who creates a new workload cluster (and some specific other users) can manage and access the workload cluster.
+
+- As an operator of a management cluster, I want the user who creates a new workload cluster to be able to give permission to other users to manage that cluster.
+
+- As an operator of a management cluster, I want to configure whether operators of workload clusters are allowed to open interactive shells onto those clusters machines.
+
+## Multi-cluster/Multi-provider
+
+### Managing Providers
+- As an operator, given I have a management cluster with at least one provider, I would like to install a new provider.
+
+- As an operator, given I have a management cluster with at least one provider, I would like to remove one of those providers and orphan any clusters provisioned by that provider.
```

Comment:
What does it mean by making a cluster "orphan"?

Reply:
It probably means that the cluster will have no Cluster API controller (and provider) reconciling it